<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NaCursorReadouts</class>
 <widget class="QWidget" name="NaCursorReadouts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>167</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: rgb(20, 20, 22);
  color: white;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 255, 255, 153);</string>
     </property>
     <property name="text">
      <string>CurF2= </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="customValue2">
     <property name="text">
      <string>0.00 dB</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="frequency2">
     <property name="text">
      <string>0000.000 MHz</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="customLabel2">
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 255, 255, 153);</string>
     </property>
     <property name="text">
      <string>CurMag2= </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="customLabel1">
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 255, 255, 153);</string>
     </property>
     <property name="text">
      <string>CurMag1= </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="frequency1">
     <property name="text">
      <string>0000.000 MHz</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 255, 255, 153);</string>
     </property>
     <property name="text">
      <string>CurF1= </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="customValue1">
     <property name="text">
      <string>0.00 dB</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="deltaLabel">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 255, 255, 153);</string>
     </property>
     <property name="text">
      <string>Î”Mag= </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="deltaValue">
     <property name="text">
      <string>0.00 dB</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
